docs/sources/alerting/manage-notifications/alertmanager.md

```diff
@@ -32,9 +32,9 @@ If you want to use a single alertmanager to receive all your Grafana, Loki, Mimi
 
 Here are two examples of when you may want to configure your own external alertmanager and send your alerts there instead of the Grafana Alertmanager:
 
-1. You may already have alertmanagers on-premise in your own Cloud infrastructure that you have set up and still want to use, because you have other alert generators, such as Prometheus.
+1. You may already have alertmanagers on-premises in your own Cloud infrastructure that you have set up and still want to use, because you have other alert generators, such as Prometheus.
```

Comment:
```suggestion
1. You may already have Alertmanagers on-premises in your own Cloud infrastructure that you have set up and still want to use, because you have other alert generators, such as Prometheus.
```